src/constants/event.ts

```diff
@@ -56,7 +56,12 @@ const EVENT_DETAIL_BUTTON = {
   showSubmitForm: '제출된 폼 보기',
   edit: '수정하기',
   delete: '삭제하기',
-  apply: '참여 신청하기',
+  apply: {
+    completed: '신청 완료',
+    soldOut: 'SOLD OUT',
+    deadLine: '신청 마감',
+    possible: '참여 신청하기',
+  },
```

Comment:
```suggestion
  APPLY: {
    COMPLETED: '신청 완료',
    SOLD_OUT: 'SOLD OUT',
    DEAD_LINE: '신청 마감',
    POSSIBLE: '참여 신청하기',
  },
```
얘네도 대문자로 써야 상수 느낌이 날 것 같네요!

Reply:
constant/event.ts의 경우에는 모두 이와 같이 적용되어 있는데,
이 부분만 이렇게 적용하나요??? 상수 부분도 리팩토링이 필요하겠군요!

Reply:
따로 상수만 리팩토링하는 브랜치에서 진행하겠습니다!